XamlControlsGallery/App.xaml

```diff
@@ -393,7 +393,6 @@
             <Style x:Key="AutoSuggestBoxStyle" TargetType="AutoSuggestBox">
```

Comment:
I think it's fine to delete this style completely, it's really not accomplishing very much at this point.

Reply:
Removed the style in the latest commit.